umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10.clj

```diff
@@ -185,7 +185,8 @@
           :let [[type subtype] (dif-util/umm-url-type->dif-umm-content-type
                                 (util/remove-nil-keys
                                  (select-keys related-url [:URLContentType :Type :Subtype])))
-                mime-type (get-in related-url [:GetService :MimeType])
+                mime-type (or (get-in related-url [:GetService :MimeType])
```

Comment:
Is there any reason to prioritize the MimeType of the GetService URL over the GetData URL (or vice versa)?

Reply:
I don't think so, unless @eereiter has different ideas.

Reply:
no